rmf_fleet_adapter/schemas/event_description__charge_battery.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/open-rmf/rmf_ros2/main/rmf_fleet_adapter/schemas/event_description__charge_battery.json",
+  "title": "Charge Battery Event",
+  "description": "Charge a robot's battery",
+  "type": "object",
+  "properties": {
```

Comment:
I think right now the battery level to which the robot charges up to is statically defined in the fleet config. In the future it would be good to specify this % in the `properties` here and have the `TaskPlanner` accept this value in the `Options` it receives before it plans. Not a blocker.